arm-helper/package.json

```diff
@@ -15,18 +19,19 @@
   "homepage": "https://github.com/HomecareHomebase/azure-bake",
   "scripts": {
     "upload": "tsc -p ./src && npm --no-git-tag-version version patch && npm publish --access public",
-    "compile": "tsc",
+    "compile": "tsc -p ./src",
     "watch": "tsc -w -p ./src"
   },
   "peerDependencies": {
-    "@azbake/core": "0.*",
+    "@azbake/core": "^0.1.17",
```

Comment:
PeerDeps need to be in the format of "x.*" vs ^format. npm versioning is a bit funky with ^ format. For peerDeps we want to support anything of X.*.* where X is the major version we are supporting. Right now, that's basically 0.* for all our peerDeps, including azure API packages. This comment should be applied to all package.json files that include peerdeps.

Reply:
Fixed it in peer dependencies. I also understand the `^` semver conventions. `^` will allow versioning that does not modify the left most set digit. In the case of a `0.0.2` then you can only update patch versions, not minor or major.